x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/create_policy_extension.test.tsx

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
I have to say those are really cool tests 👑

Reply:
mostly found in kibana tbh :)